tokens/gen.go

```diff
@@ -0,0 +1,7 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package tokens
+
+//go:generate mkdir -p "erc20"
+//go:generate abigen --bin=../contracts/build/ERC20.bin --abi=../contracts/build/ERC20.abi --type=ERC20 --pkg=erc20 --out=erc20/erc20.go
```

Comment:
Whenever I try a go generate, I get the error
Fatal: Failed to read input ABI: open ../../contracts/build/TestToken.abi: The system cannot find the path specified.
What am I doing wrong? I've tried running go install and go build but I do not see the /contracts/build folder showing up anywhere.